src/app/ReadClient.h

```diff
@@ -110,11 +110,15 @@ class ReadClient : public Messaging::ExchangeDelegate
          * receives an OnDone call to destroy the object.
          *
          * @param[in] aPath        The attribute path field in report response.
+         * @param[in] aVersion    The data version for cluster in report response. The version could be omited when
+         * EnableTagCompression in the Path field is true or the current report response don't have AttributeDataIB.
```

Comment:
nit: add spaces to align with the comments

Reply:
Done, thanks